### PR TITLE
:mortar_board: Topic representation --- Geno/pheno and TSP example 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

1. Hit genotype/phenotype again
2. Discuss difference in phenotype and genotype space
3. Show TSP example reps 

### Testing

:+1: built local, looks good

### Additional Notes

I stole your pic

### CODE REVIEW

File: site/topics/representation/representation.rst [hunk 1]
```diff
@@ -7,6 +7,203 @@ Representation
...
+
+TSP Example
+-----------
+
+* Consider the TSP example previous discussed
+
+.. figure:: ../representation/tsp_horse.png
+    :width: 333 px
+    :align: center
+    :target: https://github.com/jameshughes89/cs4XX-EvolutionaryComputation/pull/64#discussion_r1284677649
```
Comment (twentylemon) [inline site/topics/representation/representation.rst:56]: This may not work because it's a private repo. I can send you the SVG if it fails

**Review by twentylemon — APPROVED** [review]

File: site/topics/representation/representation.rst (hunk 1 above)
Comment (jameshughes89) [reply site/topics/representation/representation.rst:56]: The link is to a public repo 
image

Comment (twentylemon) [reply site/topics/representation/representation.rst:74]: Oh. Iunno why I figured this was private. I blame Dave.

Comment (jameshughes89) [reply site/topics/representation/representation.rst:74]: > I blame Dave

It's the only reasonable explanation.

**Pull request merged**